src/endpoint/endpoint.js

```diff
@@ -178,11 +177,6 @@ async function main(options = {}) {
             await listen_http(https_port_sts, https_server_sts);
             dbg.log0('Started STS HTTPS successfully');
         }
-        if (metrics_port > 0) {
```

Comment:
won't this create a bug on containerized env?

Reply:
@romayalon you are saying in containerized env there is no fork so we will miss the metrics server creation. Is that right?

Reply:
I think that forks were added specifically for non containerized env, so yes, please check that

Reply:
@romayalon  you are correct it was affecting the containerized prometheus deployment, I added the previous code for containerized flow.